### PR TITLE
Nw24000512 postgres migration

### PR DESCRIPTION
**Pull request by foresti-smeup**
## Description

Add isNumeric boolean to metadata fields passed to reload (new version 1.4.0 migrated to postgres)

Related to # (issue)

## Checklist:
- [X] There are tests regarding this feature
- [X] The code follows the Kotlin conventions (run `./gradlew ktlintCheck`)
- [X] The code passes all tests (run `./gradlew check`)
- [ ] There is a specific documentation in the `docs` directory

### CODE REVIEW

**Review by lanarimarco — CHANGES_REQUESTED** [review]
If not necessary remove `Main.kt`

File: jariko/src/Main.kt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+fun main() {
```
[the remark this answers is not in the document]
Comment (foresti-smeup) [reply jariko/src/Main.kt:1]: The best code I've written recently, and you want me to remove it?

[the remark this answers is not in the document]
Comment (lanarimarco) [reply jariko/src/Main.kt:1]: The best code I have ever seen...

[the remark this answers is not in the document]
Comment (foresti-smeup) [reply jariko/src/Main.kt]: removed

**Review by lanarimarco — APPROVED** [review]
Ok

**Pull request merged**